spec/core/account/schemas/AccountKeyDTO.yml

```diff
@@ -0,0 +1,9 @@
+type: object
+required:
+  - keyType
+  - key
+properties:
+  keyType:
+    $ref: "./KeyTypeEnum.yml"
+  message:
+    key: string
```

Comment:
i think this is wrong...? remove message for key and add key type (unsure if string or we have better defined)